package.json

```diff
@@ -68,6 +69,7 @@
         "gulp-dart-sass": "^1.0.2",
         "gulp-replace": "^1.1.3",
         "gulp-typescript": "^5.0.1",
+        "nano-staged": "^0.5.0",
```

Comment:
Why not `lint-staged` that we already familiar with?

Reply:
`nano-staged` has identical functionality and config, but is much smaller, has only one small dependency and faster. https://github.com/usmanyunusov/nano-staged#benchmarks